operators/pkg/controller/common/certificates/ca_reconcile.go

```diff
@@ -201,7 +201,7 @@ func buildCAFromSecret(caInternalSecret corev1.Secret) *CA {
 	}
 	privateKey, err := ParsePEMPrivateKey(privateKeyBytes)
 	if err != nil {
-		log.Info("Cannot parse PEM private key from CA secret, will create a new one", "err", err)
+		log.Error(err, "Cannot parse PEM private key from CA secret]")
```

Comment:
Is the `]` a typo here?
Also, I liked the "will create a new one" part.

Reply:
I had removed the "will create a new one" because this function isn't doing that. This is just returning a CA if it can, or returning nil if it can't.